sympy/core/compatibility.py

```diff
@@ -488,6 +488,14 @@ def default_sort_key(item, order=None):
     The order of terms obtained when using these keys is the order that would
     be obtained if those terms were *factors* in a product.
 
+    It is not useful to directly sort expressions based on their complexity
```

Comment:
> the See Also [should have] "ordered" listed

I think it's important to refer to the function which *can* sort by complexity. It's in the same file so the full path is not needed; it can just be added as `ordered, ` to the list that is there.